### PR TITLE
also copy 'ext', set $PATH and $PYTHONPATH correctly for Stampy

### PR DESCRIPTION
**Pull request by boegel**
overlooked in #2180

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8ed066b06b0432cab0b3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/393fbaf0e115a519007d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/13759df165522517a2dd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/40a6cb760d4ac9eff5c1 for a full test report.

File: easybuild/easyconfigs/s/Stampy/Stampy-1.0.28-intel-2015b-Python-2.7.10.eb [hunk 1]
```diff
@@ -22,11 +22,16 @@ dependencies = [
...
-files_to_copy = ['maptools.so', 'plugins', 'README.txt', 'Stampy', 'stampy.py']
+files_to_copy = ['ext', 'maptools.so', 'plugins', 'README.txt', 'Stampy', 'stampy.py']
 
 sanity_check_paths = {
     'files': ['maptools.so', 'stampy.py'],
-    'dirs': ['plugins', 'Stampy'],
+    'dirs': ['ext', 'plugins', 'Stampy'],
+}
+
+modextrapaths = {
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/Stampy/Stampy-1.0.28-intel-2015b-Python-2.7.10.eb]: might add a comment why they are empty? Seems weird for new/external people.

Comment (boegel) [inline easybuild/easyconfigs/s/Stampy/Stampy-1.0.28-intel-2015b-Python-2.7.10.eb]: done

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**